docs/docs/deploying-to-azure.md

```diff
@@ -0,0 +1,136 @@
+
+   ```bash
+   git remote add origin https://github.com/<user>/gatsby-app.git
+   git push -u origin master
+   ```
+
+Now you are ready to deploy your site!
+
+## Deploy to Azure Static Web Apps
+
+1. Sign in to the [Azure portal](https://portal.azure.com/learn.docs.microsoft.com?azure-portal=true).
+1. In the top bar, search for **Static Web Apps**.
+1. Select **Static Web Apps**.
+1. Select **New**.
+
+### Configuring settings
+
+Next, configure your new app and link it to your GitHub repository.
+
+1. Enter the **Project Details**.
+
+   | Setting          | Value                                  |
+   | ---------------- | -------------------------------------- |
+   | _Subscription_   | **Your chosen subscription**           |
+   | _Resource Group_ | _A pre-existing or new resource group_ |
+
+1. Enter the **Static Web Apps details**.
+
+   | Setting  | Value                                                                         |
+   | -------- | ----------------------------------------------------------------------------- |
+   | _Name_   | Name your app. Valid characters are `a-z` (case insensitive), `0-9`, and `_`. |
+   | _Region_ | Select Region closest to you                                                  |
+   | _SKU_    | **Free**                                                                      |
+
+1. Click the **Sign-in with GitHub** button and authenticate with GitHub.
+1. Enter the **Source Control Details**.
+
+   | Setting        | Value                                                    |
+   | -------------- | -------------------------------------------------------- |
+   | _Organization_ | Select the Organization where you created the repository |
+   | _Repository_   | **gatsby-app**                                           |
+   | _Branch_       | **master**                                               |
+
+1. Click the **Next: Build >** button to edit the build configuration.
+
+### Building your site
+
+Next, add configuration details specific to your preferred frontend framework.
+
+| Setting                 | Value         |
+| ----------------------- | ------------- |
+| _App location_          | _Leave blank_ |
+| _Api location_          | _Leave blank_ |
+| _App artifact location_ | **build**     |
+
+Click the **Review + create** button.
+
+Continue to create the application.
+
+1. Click the **Create** button
+
+1. Once the deployment is complete, click the **Go to resource** button
+
+### Review the GitHub Action
+
+At this stage, your Static Web Apps instance is created in Azure but your app is not yet deployed. The GitHub Action that Azure creates in your repository will run automatically to perform the first build and deployment of your app, but it takes a couple of minutes to finish.
+
+You can check the status of your build and deploy action by clicking a big blue border at the top of your main window.
```

Comment:
```suggestion
You can check the status of your build and deploy action by clicking the blue information banner at the top of your main window.
```

This could still be misread as referring to the main navbar at the top of the screen.